bcsv.py

```diff
@@ -8,7 +8,7 @@
 
 def printHex(s):
     for i in range(0, len(s), 4):
-        print ''.join(['%02X'%ord(x) for x in s[i:min(i+4, len(s))]]),
+        print(''.join(['%02X'%ord(x) for x in s[i:min(i+4, len(s))]]),)
```

Comment:
The trailing comma has meaning in Python 2 print statements, which you have not replicated in the Python 3 print call.